internal/reconciler/metricpipeline/config_builder.go

```diff
@@ -59,6 +64,13 @@ func makeOtelCollectorConfig(ctx context.Context, c client.Reader, pipelines []v
 	}, allVars, nil
 }
 
+func getProcessorAliases(output *v1alpha1.MetricPipelineOutput) []string {
+	if output.ToDelta {
+		return []string{"cumulativetodelta", "memory_limiter", "k8sattributes", "resource", "batch"}
```

Comment:
Do some investigation if the order of the processors matter. Iirc, the `memory_limiter` should be the first processor.